src/styles/components/_search-bar.scss

```diff
@@ -29,6 +29,10 @@
 .c-search-bar__value-placeholder {
 	color: $neutral;
 	margin-right: 1.4rem;
+
+	@include respond-to(md) {
+		display: none;
+	}
```

Comment:
Verbergt dit elke placeholder? If so, is het geen algemene regel van elke search-bar instantie https://xd.adobe.com/view/86b0be3f-d968-4a98-7784-d5d00cd42f98-f081/screen/8ad8407d-d973-424b-8bc3-e04d7db71123/specs/

Reply:
Dit is enkel voor de placeholder die verschijnt als er een waarde ingevuld is:
https://xd.adobe.com/view/86b0be3f-d968-4a98-7784-d5d00cd42f98-f081/screen/c3bdc060-e5c9-4273-9d56-b07f47570969 
vs 
https://xd.adobe.com/view/86b0be3f-d968-4a98-7784-d5d00cd42f98-f081/screen/52e67350-c6a8-4f29-bd6f-3940196436ac
De normale placeholder als er geen waarde is blijft altijd staan